src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -544,11 +554,21 @@ unprivileged_init( fd_topo_t *      topo,
   ctx->replay->acc_mgr      = ctx->slot_ctx->acc_mgr;
   ctx->replay->funk         = funk;
 
-  ctx->tpool = fd_tpool_init( ctx->tpool_mem, 1 );
-  ctx->max_workers = 1;
+  ctx->max_workers = tile->replay.tpool_thread_count;
```

Comment:
+1 to include the main thread, unless the thread count already does

Reply:
it includes the main thread this way